src/Microsoft.AspNetCore.DataProtection/NullLoggerFactory.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.DataProtection
+{
+    /// <summary>
+    /// An <see cref="ILoggerFactory"/> that provides a logger that no ops.
+    /// </summary>
+    public class NullLoggerFactory : ILoggerFactory
```

Comment:
And... I looked at it some more, and is this used only in tests to begin with? I don't think I see it used at all in `src`, in which case at the very very least least it can move to the `test` folder as a utility library. But maybe it really should go into the Logging repo anywya.